BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -34,10 +35,20 @@ extension Store {
     
     init?(json: NSDictionary) {
         
-        guard let storeName = json["store_name"] as? String,
-            let addressJSON = json["address"] as? NSDictionary,
-            let storeImageString = json["main_image_url"] as? String,
-            let storeImageURL = URL(string: storeImageString)
+        let storeImageString = json ["main_image_url"] as? String
+        
+        if storeImageString == nil {
+
+            storeImageURL = URL(string:"https://s3.amazonaws.com/ODNUploads/5404f487678f1532dfdbcc6479placeholder_food_item_2.png")
+        }
+        else {
+            
+            storeImageURL = URL(string: storeImageString ?? "")
```

Comment:
You should do optional binding so that `storeImageString` isn't an optional within this block.